### PR TITLE
JSUI-2827 Fixed plural label for `CategoryValue`

### PR DESCRIPTION
**Pull request by btaillon**
https://coveord.atlassian.net/browse/JSUI-2827

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: src/ui/CategoryFacet/CategoryValue.ts [hunk 1]
```diff
@@ -83,7 +83,8 @@ export class CategoryValue implements CategoryValueParent {
     const element = $$(this.element.find('.coveo-category-facet-value-label'));
     element.addClass('coveo-selectable');
 
-    const countLabel = l('ResultCount', this.categoryValueDescriptor.count.toString());
+    const count = this.categoryValueDescriptor.count;
+    const countLabel = l('ResultCount', count.toString(), count);
```
Comment (samisayegh) [inline src/ui/CategoryFacet/CategoryValue.ts]: Though it's a small change, I prefer to include a unit test to document and enforce the Deque recommendation that we paid for.

Comment (btaillon) [reply src/ui/CategoryFacet/CategoryValue.ts]: Good point, I'll add unit tests for it.

File: unitTests/ui/CategoryFacet/CategoryValueTest.ts [hunk 2]
```diff
@@ -117,13 +117,27 @@ export function CategoryValueTest() {
...
+          const labelAttribute = labelElement.attributes['aria-label'];
+          expect(labelAttribute.value).toContain(caption);
+        });
+
+        it('adds a label containing the results count', () => {
+          const labelAttribute = labelElement.attributes['aria-label'];
+          expect(labelAttribute.value).toContain(categoryValueDescriptor.count);
+        });
+
+        it("doesn't add plural tags", () => {
```
Comment (samisayegh) [inline unitTests/ui/CategoryFacet/CategoryValueTest.ts]: Should we have a test when `count >= 2` that checks that the plural form is being used?

Comment (btaillon) [reply unitTests/ui/CategoryFacet/CategoryValueTest.ts]: Perhaps, I'll add a test for it.

**Pull request merged**